backend/sirius-web-diagrams/src/main/java/org/eclipse/sirius/web/diagrams/ViewCreationRequest.java

```diff
@@ -37,11 +37,11 @@ private ViewCreationRequest() {
     }
 
     /**
-     * Specify the identifier of the parent view.
+     * Specify the identifier of the parent view. (null when the parent is the diagram itself)
```

Comment:
This can be removed, since the id is a string, null is not a valid value and technically, the code does not allow null

Reply:
Done